src/App.js

```diff
@@ -200,10 +216,12 @@ export default function App() {
       filterFrequency,
       filterType,
       resonance,
+      ampEnvelope,
       reverbDuration,
       phaserDuration,
     };
     savePatch(savedPatch);
+    console.log(savedPatch);
```

Comment:
Not sure if you want to log this for the users

Reply:
No, that was for me to check if the amp.properties actually do save.